quickfixj-core/src/main/java/quickfix/mina/initiator/InitiatorProxyIoHandler.java

```diff
@@ -61,8 +61,8 @@ public void exceptionCaught(IoSession ioSession, Throwable cause) throws Excepti
 
     @Override
     public void proxySessionOpened(IoSession ioSession) throws Exception {
-        if (this.sslFilter != null) {
-            this.sslFilter.initiateHandshake(ioSession);
-        }
+//        if (this.sslFilter != null) {
```

Comment:
Not sure if this is even required, mina proxy connector should be handling handshake...

Reply:
I cannot tell for sure TBH.

Reply:
For the time being I've removed the whole SslFilter from this class. Maybe once there is some working proxy integration we can check if it actually works.